lib/punt/cluster.go

```diff
@@ -117,6 +123,17 @@ func (c *Cluster) Run() error {
 	return nil
 }
 
+func (c *Cluster) gcIndexes() {
+	ticker := time.NewTicker(time.Second * 5)
```

Comment:
How often do we create new indexes? Is it worth deleting them every 5 seconds?

Reply:
Ah debug code :sweat_smile: